data/transactions/logic/eval.go

```diff
@@ -272,7 +290,8 @@ type evalContext struct {
 
 	stack     []stackValue
 	callstack []int
-	program   []byte // txn.Lsig.Logic ?
+	subtxn    transactions.Transaction // place to build for tx_submit
```

Comment:
tx_begin and tx_field work together to populate subtxn. tx_submit Performs()s it.

tx_submit should also clear it.

Reply:
One at a time.  After `tx_perform` you can `tx_begin` again, set some fields, and `tx_perform` again.  If/when we allow the construction of groups, this field would have to become a slice, to hold onto multiple pending txns at once.